src/containers/DetailScreen.js

```diff
@@ -1,12 +1,5 @@
 import React, { PureComponent } from 'react';
-import {
-  FlatList,
-  Image,
-  ScrollView,
-  StyleSheet,
-  Text,
-  View,
-} from 'react-native';
+import { FlatList, Image, StyleSheet, Text, View, ScrollView } from 'react-native';
```

Comment:
When there are so many import items, it's nice to break it out into individual lines so that it's easier to scan.  I put the arbitrary limit at 3 items side by side usually :P)

Reply:
```import { FlatList, Image, StyleSheet,
  Text, View, ScrollView } from 'react-native';```

Reply:
?

Reply:
Or just line it all back out
```
FlatList,		
Image,		
ScrollView,		
StyleSheet,		
Text,		
View,
```
Another advantage of having it all lined out like this is that we can alphabetize it

Reply:
More specifically, 
if n<4, then inline is fine
if n>=4, then break out all the imports into individual lines.